python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -222,6 +222,8 @@ def __post_init__(self) -> None:
             raise NotImplementedError(
                 "Read from cloud storage"
             )  # pragma: no cover; no test yet
+        if any(p.startswith("https://") for p in self.paths):
```

Comment:
What about `http://` or other protocols?

Reply:
The only way this occurs is because the user wrote `scan_foo("hf://some_path/")` and it is expanded before we see it into `https://huggingface..../some_path`.

So I could tighten up to match the full path.

Reply:
I had a broader brush to catch other URL-like things too.

Reply:
This is probably fine. If no https:// URLs are supported then let’s not make it specific to HuggingFace.